src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/StageAppYamlTask.java

```diff
@@ -19,12 +19,17 @@
 
 import com.google.cloud.tools.appengine.AppEngineException;
 import com.google.cloud.tools.appengine.operations.AppYamlProjectStaging;
+import javax.inject.Inject;
 import org.gradle.api.DefaultTask;
+import org.gradle.api.internal.file.FileOperations;
```

Comment:
Please do not import `org.gradle.api.internal` packages. Those are not meant for general use in plugin code
```suggestion
import org.gradle.api.file.FileSystemOperations;
```